go.mod

```
@@ -1,7 +1,8 @@
 module github.com/devcyclehq/terraform-provider-devcycle
 
-go 1.19
-toolchain go1.24.1
+go 1.23.0
```

this can be fix by doing the following
```suggestion
go 1.23
```